File: src/main/java/com/procurement/mdm/model/entity/Language.java
package com.procurement.mdm.model.entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@Entity
@Table(name = "languages")
public class Language {

    @Id
    @Column(name = "id")
    private Long id;

    @Column(name = "s_no")
    private Integer sNo;

    @Column(name = "name")
    private String name;

    @Column(name = "family")
    private String family;

    @Column(name = "iso_639_1")
    private String iso6391;

    @Column(name = "description")
    private String description;
}

File: src/main/kotlin/com/procurement/mdm/model/entity/Language.kt
package com.procurement.mdm.model.entity

import com.fasterxml.jackson.annotation.JsonProperty
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "language")
data class Language(

        @Id
        @Column(name = "code", length = 2)
        val code: String = "",

        @Column(name = "name")
        val name: String = "",

        @Column(name = "description")
        val description: String = ""
)

data class LanguageDto(

        @JsonProperty("code")
        val code: String?,

        @JsonProperty("name")
        val name: String?
)

fun List<Language>.getItems(): List<LanguageDto> =
        this.asSequence().map { LanguageDto(code = it.code, name = it.name) }.toList()
